docs/.vuepress/config.js

```diff
@@ -1,13 +1,41 @@
 'use strict';
 
 const { readFileSync } = require('fs');
+const markdownItForInline = require('markdown-it-for-inline');
+const path = require('path');
+
+const docsMetaData = require('../dump.json');
 const generateSidebar = require('../../src/generate-sidebar.js');
 const manifest = require('./public/manifest.json');
 
+const derp = url => {
+  const urlMatch = url.match(/paper\.dropbox\.com\/doc\/.+--\S{26}-(\w{21})/);
+
+  if (urlMatch) {
+    const docMatch = docsMetaData.find(doc => doc.id === urlMatch[1]);
+    console.log(urlMatch, docMatch);
+    return docMatch ? docMatch.location : false;
+  }
+
+  return false;
+};
+
 module.exports = {
   title: 'Playbook',
   themeConfig: {
-    sidebar: generateSidebar(JSON.parse(readFileSync('docs/dump.json'))),
+    sidebar: generateSidebar(docsMetaData),
+  },
+  extendMarkdown: markdown => {
+    markdown.use(markdownItForInline, 'internal-link', 'link_open', (tokens, idx) => {
```

Comment:
make this a bit easier to understand?
`idx` -> `index`
`hi` -> ?
put `path.parse(hi)` into a meaningful variable as you use it multiple times?
(`const doc = path.parse(hi);` and `const dir = path.parse(doc.dir);`?)